jdaviz/configs/mosviz/tests/test_helper.py

```diff
@@ -208,6 +208,18 @@ def test_load_image(mosviz_app, image):
     assert list(data.keys())[0] == f"{label} 0"
 
 
+def load_single_image_multi_spec(mosviz_app, image, spectrum1d, spectrum2d):
+    spectra1d = [spectrum1d for _ in range(3)]
+    spectra2d = [spectrum2d for _ in range(3)]
+
+    mosviz_app.load_data(spectra1d, spectra2d, images=image)
+
+    assert len(mosviz.app.data_collection) == 3
+
+    qtable = mosviz.to_table()
+    assert np.all(qtable["Images"] == "Shared Image")
```

Comment:
Do we also need to check the length of `qtable`?

Reply:
Wouldn't hurt, but really maybe I should add a separate test of `to_table`. Apparently there's only one for `to_csv` (probably my fault).